src/platform/windows.rs

```diff
@@ -79,11 +76,11 @@ impl<T> EventLoopExtWindows for EventLoop<T> {
 /// Additional methods on `Window` that are specific to Windows.
 pub trait WindowExtWindows {
   /// Returns the HINSTANCE of the window
-  fn hinstance(&self) -> *mut libc::c_void;
+  fn hinstance(&self) -> HINSTANCE;
```

Comment:
I agree, removing the reexported types from the public interface is cleaner. How about the internal Rust type that windows-rs uses instead? In this example, I think that would be `isize`. Or TAO could export its own opaque tuple types for cases like this, so it's still strongly typed.

Reply:
Exporting the inner type, i.e. `isize` is good enough I guess, but since other platforms other than windows with similar methods are exporting `*mut libc::c_void`, we can cast `isize` to it for consistency across the code 
base.